src/constants/languages.ts

```diff
@@ -36,6 +36,7 @@ export const LANGUAGE_NAME_MAP: { readonly [key in Language]: string } = {
 export const TEMP_SUPPORTED_LANGUAGES: ReadonlySet<Language> = new Set([
   Language.Python,
   Language.TypeScript,
+  Language.Java,
```

Comment:
We have a separate PR only for adding java support when rdy here https://github.com/cdklabs/construct-hub-webapp/pull/384/files

Reply:
Ahh good callout, I did not mean to commit that here